src/main/java/com/denizenscript/clientizen/util/CexCommand.java

```diff
@@ -0,0 +1,211 @@
+package com.denizenscript.clientizen.util;
+
+import com.denizenscript.denizencore.DenizenCore;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+import com.denizenscript.denizencore.objects.ObjectFetcher;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.ObjectType;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagManager;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.denizencore.utilities.ExCommandHelper;
+import com.denizenscript.denizencore.utilities.debugging.DebugInternals;
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.suggestion.SuggestionProvider;
+import com.mojang.brigadier.suggestion.Suggestions;
+import com.mojang.brigadier.suggestion.SuggestionsBuilder;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.text.Text;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import static com.mojang.brigadier.arguments.StringArgumentType.getString;
+import static com.mojang.brigadier.arguments.StringArgumentType.greedyString;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.argument;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class CexCommand {
+
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher) {
+        final DenizenCommandsSuggestionProvider suggestionProvider = new DenizenCommandsSuggestionProvider();
+        dispatcher.register(literal("cex").then(argument("command", greedyString())
+                .suggests(suggestionProvider)
+                .executes(context -> execute(context, false))));
+        dispatcher.register(literal("cexq").then(argument("command", greedyString())
+                .suggests(suggestionProvider)
+                .executes(context -> execute(context, true))));
+    }
+
+    public static int execute(CommandContext<FabricClientCommandSource> context, boolean silent) {
+        ExCommandHelper.runString("CEXCOMMAND", getString(context, "command"), null, silent ? null : queue -> {
+            queue.debugOutput = debug -> context.getSource().sendFeedback(Text.literal(debug.replace("<FORCE_ALIGN>", "")));
```

Comment:
I've been seeing the removal of special codes like `<FORCE_ALIGN>` in a few places, maybe we could extract these to the utility methods?

Reply:
I don't think it's worth a util method tbh, it's only `<FORCE_ALIGN>` and it's only done in places that handle debug output, which it's unlikely we'll have too many of